portal/templates/utilisation-report-service/record-correction/correction-sent.njk

```diff
@@ -0,0 +1,24 @@
+{% extends "index.njk" %}
+
+{% import "../_macros/header-only-panel.njk" as headerOnlyPanel %}
+{% import "../_macros/email-confirmation-list.njk" as emailConfirmationList %}
+
+{% block pageTitle %}Correction sent{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ headerOnlyPanel.render({
+        titleText: formattedReportPeriod + ' record correction sent to UKEF'
+      }) }}
+
+      {{ emailConfirmationList.render({
+        emails: sentToEmails
+      }) }}
+
+      <p class="govuk-body">UKEF will be notified via email that a record has been updated.</p>
```

Comment:
data-cy please

Reply:
It doesn't need one currently because the e2e-tests have not been added yet. I will add one when I write the e2e-tests to avoid having selectors that are never used cluttering the code.

Reply:
the continue button below already has a data-cy and this selector will be used in the future.  Saves the hassle of forgetting

Reply:
Continue button data-cy is already used in the component tests

Reply:
I will be the person adding the e2e-tests and I find it easier to do it this way